src/modules/layout/components/WalletBalance/WalletBalance.tsx

```diff
@@ -60,6 +72,16 @@ export const WalletBalance = ({
             {ankr.decimalPlaces(DEFAULT_FIXED).toFormat()}
           </Typography>
         )}
+
+        {avax && (
+          <Typography
+            color="textSecondary"
+            className={classNames(classes.item, classes.avax)}
+            title="Avax"
```

Comment:
```suggestion
            title={t('foobar.avax')}
```

All literals must be translated 🤷‍♂️